.travis.yml

```diff
@@ -4,9 +4,9 @@ node_js:
 notifications:
   email: false
 before_install:
-- npm install --global lerna@3.2.1 nyc mocha typescript@2.7.2
```

Comment:
I have to ask, what happened with `nyc`and `mocha`?

Reply:
`mocha` is already a devDependency, `nyc` gets installed when `npm install` is ran in `.\tools\package.json`